### DISCUSSION TITLE
Github pages deployment returning 404 for css and js

### DISCUSSION THREAD

**Discussion by ThatNerdUKnow** [community/community#21591] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve written a vue application and am using a github action to create a branch of only the contents of only the ./dist folder.<br>
My index.html loads just fine but any resources requested from index.html like any javascript or css fails with a 404. I’m not sure what to do here<br>
my repository is <a href="https://github.com/Withenex/DnD-BattleTracker" class="inline-onebox" rel="noopener nofollow ugc">GitHub - Withenex/DnD-BattleTracker</a></p>

**Comment by gojimmypi** [comment, marked as answer]
<p>I wonder if you are also missing a <code>baseurl</code> reference, similar to my problem as noted <a href="https://github.community/t/github-page-jekyll-looks-very-different-from-local-from-the-very-beginning/10816/5">here</a>?</p>

**Comment by ThatNerdUKnow** [comment]
<p>That did the trick! all I had to do was set publicPath to “” in my vue.config.js file. This changes the file references to use relative paths in the build process as I understand it. Thanks for the help!</p>
